openml/datasets/dataset.py

```diff
@@ -30,8 +30,6 @@ class OpenMLDataset(object):
         Name of the dataset.
     description : str
         Description of the dataset.
-    format : str
```

Comment:
You need to keep the documentation for the new parameter.
If it is going to be deprecated, we can use `.. deprecated:: xxx` from sphinx

Reply:
You are right, in a previous discussion we decided to remove format and I removed it from the documentation, it should be there along with the message.

Reply:
Now that I took a second look at this, format in my understanding is written as the attribute of the class, the attribute of the class is still there and it is not changed. However I changed the name of the parameter given to the init because it shadowed the builtin format. So a deprecation warning should do the trick inside the init.